format_all.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
Do not show this script in the root directory. It is not that important. Move it to the `tools` directory.